src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/DeleteReferenceValidation.feature

```diff
@@ -1,150 +1,120 @@
+                  | schoolYear | currentSchoolYear | schoolYearDescription |
+                  | 2022       | false             | 2021-2022             |
+             And the system has these "calendars"
+                  | calendarCode   | schoolReference         | schoolYearTypeReference | calendarTypeDescriptor                                  |
+                  | "2010605675"   | {"schoolId":255901107}  | {"schoolYear":2022}     | uri://ed-fi.org/CalendarTypeDescriptor#Student Specific |
+             When a DELETE request is made to referenced resource "/ed-fi/schoolYearTypes/{id}"
              Then it should respond with 409
               And the response body is
                   """
                   {
-                      "detail": "The requested action cannot be performed because this item is referenced by an existing 'Calendar' item.",
+                      "detail": "The requested action cannot be performed because this item is referenced by existing Calendar item(s).",
                       "type": "urn:ed-fi:api:data-conflict:dependent-item-exists",
                       "title": "Dependent Item Exists",
                       "status": 409,
-                      "correlationId": null
+                      "correlationId": null,
+                      "validationErrors": null,
+                      "errors": null
                   }
                   """
 
         @ignore
         Scenario: 02 Ensure clients cannot delete a descriptor that is used by another item
-            Given a POST request is made to "/ed-fi/educationOrganizationCategoryDescriptors" with
-                  """
-                  {
-                      "codeValue": "School",
-                      "shortDescription": "School",
-                      "namespace": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor"
-                  }
-                  """
-             When a DELETE request is made to "/ed-fi/educationOrganizationCategoryDescriptors/{id}"
+            Given Education Organization Category Descriptor School
+             When a DELETE request is made to "ed-fi/educationOrganizationCategoryDescriptors/{id}"
              Then it should respond with 409
               And the response body is
                   """
                   {
-                      "detail": "The requested action cannot be performed because this item is referenced by an existing 'School' item.",
+                      "detail": "The requested action cannot be performed because this item is referenced by existing School item(s).",
                       "type": "urn:ed-fi:api:data-conflict:dependent-item-exists",
                       "title": "Dependent Item Exists",
                       "status": 409,
-                      "correlationId": null
+                      "correlationId": null,
+                      "validationErrors": null,
+                      "errors": null
```

Comment:
Since `validationErrors` and `errors` are arrays, it would be better to return `[]` instead of `null`. It is unusual to have a JSON document with a `null` value.